karapace/schema_registry_apis.py

```diff
@@ -114,29 +128,30 @@ def _create_master_coordinator(self):
         self.mc = MasterCoordinator(config=self.config)
         self.mc.start()
 
-    def _subject_get(self, subject, content_type):
+    def _subject_get(self, subject, content_type, include_deleted=False):
```

Comment:
Please add type annotation. I would suggest that we always add type annotations to functions that we modify. Eventually we will have the whole codebase typed.